terraform/constellation-cluster/install-constellation.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

more portable :) Different Unixes put bash in different locations